tst/providers/MeetingProvider/index.test.tsx

```diff
@@ -18,17 +19,21 @@ import { MeetingManagerJoinOptions } from '../../../src/providers/MeetingProvide
 import '@testing-library/jest-dom';
 
 import { act, renderHook } from '@testing-library/react';
-import React from 'react';
 
-import { MeetingProvider, useMeetingManager } from '../../../src/providers/MeetingProvider';
+import {
+  MeetingProvider,
+  useMeetingManager,
+} from '../../../src/providers/MeetingProvider';
 
 describe('Meeting Provider', () => {
   it('events are received correctly', async () => {
     // Mock the user agent to ensure the MeetingManager event subscription is
     // set up. Otherwise, the amazon-chime-sdk-js will not detect a valid
     // browser to use.
     const userAgentGet = jest.spyOn(navigator, 'userAgent', 'get');
-    userAgentGet.mockReturnValue('Chrome/96.0');
+    userAgentGet.mockReturnValue(
+      'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36'
```

Comment:
Any reason why this particular value is being used? maybe add a comment here so that other engineers know.

Reply:
This is just for testing. Any user agent works here. The reason for this change is that the previous mockValue `'Chrome/96.0'` is partial, unit test reports error when parsing this user agent. So I just replaced it with a complete example.